LIBFCREPO-1317: Adding standard predicate csv

aguilarm-umd opened the pull request:
https://umd-dit.atlassian.net/browse/LIBFCREPO-1317

aguilarm-umd committed:
LIBFCREPO-1317: Adding standard predicate csv

https://umd-dit.atlassian.net/browse/LIBFCREPO-1317

peichman-umd commented on predicate.csv:
```diff
@@ -0,0 +1,6 @@
+predicate,object_type
+rdfs:label,Literal
+owl:sameAs,URIRef
+rdfs:subClassOf, URIRef
```
There are extraneous leading spaces in the object_type column of these last three lines, which caused the `load_predicates` command to incorrectly assume a predicate that was already in the database was not.

To fix, remove the extra spaces from this file. It might also be a good idea to update the `load_predicates` command to run `.strip()` on the raw values it gets from the CSV file, in order to guard against this sort of accidental extra spaces in the future.

peichman-umd reviewed (changes requested):
Some CSV formatting fixes: one in the actual CSV file, and one to the loader to help guard against stray whitespace errors.

aguilarm-umd committed:
LIBFCREPO-1317: Updating csv file and calling strip

Fixed spacing issue in the csv and also added call to strip when
predicate and object_type are used

https://umd-dit.atlassian.net/browse/LIBFCREPO-1317
